iris-GmbH/meta-iris-base
# Feature/migl/rdphoen 1281 identity dev crt

**Test**

1. Check if key/cert is in uuu folder:
     - irma6-maintenance-uuu/sensor_identity_snakeoil.crt
     - irma6-maintenance-uuu/sensor_identity_snakeoil.key
2. Check if sensor.key and sensor.crt is in /mnt/iris/identity
3. Check if sensor.crt and sensor.key are overwritten during provisioning

**CI:** https://gitlab.devops.defra01.iris-sensing.net/MIRROR/iris-kas/-/pipelines/2008
